README.md

```diff
@@ -198,15 +199,18 @@ Currently, the following clients are available for use:
 - `ANDROID_MUSIC`
   - ✔ Opus formats.
   - ❌ No playlist/livestream support.
+- `ANDROID_VR`
+  - ✔ Opus formats.
+  - ❌ No 'YouTube Kids' video support.
```

Comment:
I don't expect this to matter for most people but I've mentioned it just for the sake of completeness.

Reply:
I wouldn't bother mentioning this to be fair. "Kids" is not something YTS deliberately aims to support.